doorman/api.py

```diff
@@ -163,6 +164,7 @@ def logger(node=None):
 
     elif log_type == 'result':
         log_tee.handle_result(data, host_identifier=node.host_identifier)
+        analyze_result.delay(data, node.to_dict())
```

Comment:
For some reason, if I comment this line out, the tests will pass locally.  When this is called at all (regardless of the location in this function), tests will fail with an invalid count.  It's too late for me to figure out why this is right now 😴